### PR TITLE
fix(api)!: correct model types and network policy docs

### PR DESCRIPTION
**Pull request by markstuart-oai**
## Summary

Removes the unsupported gpt-5.1-mini model from typed model choices and clarifies network policy and Realtime documentation. The model was never available; removing its typed choices is an intentional breaking change.

## Changes

- Remove gpt-5.1-mini from typed model choices
- Document version-specific network defaults and the difference between omitting a policy and resetting it with null
- Identify conversation.item.added and conversation.item.done in Realtime item-creation documentation
- Clarify recovery and completion semantics for agent session cancellation and deletion

BEGIN_COMMIT_OVERRIDE
fix(api): correct model types and network policy docs

Release-As: 3.17.0
END_COMMIT_OVERRIDE

### CODE REVIEW

File: src/openai/types/realtime/conversation_item_create_event.py [hunk 1]
```diff
@@ -16,8 +16,9 @@ class ConversationItemCreateEvent(BaseModel):
     "history" of the conversation and to add new items mid-stream, but has the
     current limitation that it cannot populate assistant audio messages.
 
-    If successful, the server will respond with a `conversation.item.created`
-    event, otherwise an `error` event will be sent.
+    If successful, the server will emit a `conversation.item.added` event and,
+    when the item is finalized, a `conversation.item.done` event. Otherwise, an
```
[the remark this answers is not in the document]
Comment (markstuart-oai) [reply src/openai/types/realtime/conversation_item_create_event.py:20]: The current GA wording is intentional. The [published deprecation notice](https://developers.openai.com/api/docs/deprecations#2025-09-15-realtime-api-beta) confirms that the Realtime beta interface was removed on May 12, 2026. The [current conversation guide](https://developers.openai.com/api/docs/guides/realtime-conversations) lists `conversation.item.added` and `conversation.item.done` for the GA item-creation lifecycle.

The retained `ConversationItemCreatedEvent` model and union member predate this PR; keeping those Python types does not establish that the current OpenAI endpoint emits the retired beta event. This PR preserves those types and the separate beta documentation, so it does not remove parser compatibility. Adding `conversation.item.created` here as a current supported response would make the GA guidance inaccurate.

I also checked the service's existing GA item-create test and event conversion: they assert one added event, one done event, and zero legacy created events. No new live probe or test execution is claimed. Resolving this finding as inapplicable to the current GA contract; the historical model documentation is separate scope.

File: src/openai/types/beta/agents/environments/template_create_params.py [hunk 2]
```diff
@@ -55,8 +55,7 @@ class Network(TypedDict, total=False):
     access: Required[Literal["enabled", "disabled", "restricted"]]
     """The environment's network access mode.
 
-    - `enabled` - Allows unrestricted network access, matching an omitted network
-      policy.
+    - `enabled` - Allows unrestricted network access.
```
[the remark this answers is not in the document]
Comment (markstuart-oai) [reply src/openai/types/beta/agents/environments/template_create_params.py:64]: Confirmed: the generated field documentation loses the operation-specific omission/reset description that is already present in this PR's OpenAPI input. For new alpha/beta environments without an inherited template policy, an omitted network policy enables unrestricted access; GA uses disabled. Template updates differ: omission preserves the existing policy, while null resets to the version-specific default.

The loss occurs when generation collapses the nullable reference and selects the shared NetworkPolicyParam description instead of the field description. The same narrow documentation regression appears in the TypeScript, Java, and Ruby counterparts; the Go counterpart did not remove this default text. Runtime defaults, request serialization, and type shapes are unchanged.

This finding remains open. I am scoping the maintained generator correction before changing a shared description rule; restoring the old generic enum claim would incorrectly describe template updates and GA defaults.

**Review by jbeckwith-oai — COMMENTED** [review]
Reviewed the complete 12-file diff at `741e1f84452ea384dfcf51f4b9162ccf60d89703` for source-contract accuracy, compatibility, and the networking documentation implications.

The non-documentation Python AST is unchanged. The two test files contain exactly eight expected `context_management` example corrections (`type: "compaction"`); the API-reference delta matches the corresponding public OpenAPI changes, and its recorded hash is valid. GitHub reports a successful exact-head check rollup. I ran read-only AST/source/reference checks, not the full SDK test suite or a live backend probe.

I found no additional in-scope issue beyond the [existing unresolved network-field documentation finding](https://github.com/openai/openai-python/pull/3896#discussion_r4044971434). That finding still applies to this head; I have not opened a duplicate inline thread. The branch also needs its current merge conflict reconciled and the updated head revalidated before landing.

**Review by dpiet-oai — APPROVED** [review]
